src/datreant/core/collections.py

```diff
@@ -148,32 +148,34 @@ def __getitem__(self, index):
         select out members.
 
         """
+        memberlist = self._list()
```

Comment:
I'd be in favor of this. Now that we've gotten rid of Group in #130, it would be a lot easier to put more things into ``datreant.core.collections.CollectionMixin``, since ``MemberBundle`` no longer needs to be supported by this. The only things that use that mixin now are ``Bundle`` and ``View``.